atomix/cluster/src/test/java/io/atomix/raft/RaftStartupConsistencyCheckTest.java

```diff
@@ -73,4 +75,36 @@ public void shouldNotFailRestartIfFollowerCrashedBeforeCommittingSnapshot() thro
                 assertThat(raftRule.getPersistedSnapshotStore(followerId).getCurrentSnapshotIndex())
                     .isEqualTo(snapshotIndex));
   }
+
+  @Test // regression test for https://github.com/camunda/zeebe/issues/14367
```

Comment:
🔧 Didn't you had a annotation for regression tests or doesn't this work here?

Reply:
It's only for junit 5 :disappointed: